### PR TITLE
Add multiple feature with the same type but different values to product

### PR DESCRIPTION
**Pull request by fatmaBouchekoua**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | If you create a product which has the same feature twice when you reload the page the first one disappears. This PR add the possibility to create feature more than once.
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-858
| How to test?  | Reinstall Prestashop, then add the same feature to a product more than once in the BO, Go to the FO and check the product details you will see all features in the same table line.

### CODE REVIEW

File: controllers/admin/AdminProductsController.php [hunk 1]
```diff
@@ -867,10 +867,11 @@ public function processFeatures($id_product = null)
             // add new objects
             $languages = Language::getLanguages(false);
             foreach ($_POST as $key => $val) {
-                if (preg_match('/^feature_([0-9]+)_value/i', $key, $match)) {
+                if (preg_match('/^feature_([0-9]+)_([0-9]+|)_value/i', $key, $match)) {
                     if ($val) {
                         $product->addFeaturesToDB($match[1], $val);
                     } else {
+                        dump($this->checkFeatures($languages, $match[1]));
```
Comment (aleeks) [inline controllers/admin/AdminProductsController.php]: Hello, dump here!

**Review by aleeks — CHANGES_REQUESTED** [review]

File: install-dev/data/db_structure.sql [hunk 2]
```diff
@@ -741,7 +741,7 @@ CREATE TABLE `PREFIX_feature_product` (
   `id_feature` int(10) unsigned NOT NULL,
   `id_product` int(10) unsigned NOT NULL,
   `id_feature_value` int(10) unsigned NOT NULL,
-  PRIMARY KEY (`id_feature`,`id_product`),
+  PRIMARY KEY (`id_feature`,`id_product`,`id_feature_value`),
```
Comment (aleeks) [inline install-dev/data/db_structure.sql]: Need to update in 1.7.0.0.sql file

**Review by aleeks — CHANGES_REQUESTED** [review]

**Comment by xBorderie** [comment]
I think you should try and remove the Merge commit -- maybe git-squash it?

**Comment by fatmaBouchekoua** [comment]
See #6907

**Pull request closed without merging**